src/common/dataHooks/useDictionaryEntry.js

```diff
@@ -14,11 +14,13 @@ import {
   entryForApi,
 } from 'common/dataAdaptors/dictionaryAdaptors'
 
-export function useDictionaryEntry({ id, edit = false }) {
-  const { sitename } = useParams()
+export function useDictionaryEntry({ id, sitename, edit = false }) {
+  const { paramsSitename } = useParams()
```

Comment:
The useParams variable names are set at a route level, in this case in `AppContainer.js` for all it's child components. If you need to change the name inside a component you will need to give it an alias using the syntax that you used in DictionaryDetailContainer 
`const { sitename: paramsSitename } = useParams()`